projects/admin-portal/src/app/components/request/request.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="buttons">
+  <button class="button" (click)="backToRoom()">Back to room #{{roomNumber}}</button>
+  <button class="button" (click)="backToFloor()">Back to floor #{{floorNumber}}</button>
+</div>  
+<div class="request" >
+  <div class="request__info">
+    <p class="request__static-text">Request <b class="request__info-text">#{{requestInfo.id}}</b></p>
+    <p class="request__static-text">Status - <b class="request__info-text">{{requestInfo.status | requestStatus}}</b></p>
+    <p class="request__static-text">Request Owner - <b class="request__info-text">{{requestInfo.requestOwner}}</b></p>
+    <p class="request__static-text">Open at <b class="request__info-text">{{requestInfo.openDate | date:'short'}}</b></p>
+  </div>
+  <div class="request__comments comments">
+    <div class="comments__comment-form comment-form">
+      <textarea [formControl]="newComment" class="comment-form__text"></textarea>
+      <p class="comment-form__title">Post comment as:</p>
+      <div class="comment-form__buttons" *ngIf="requestInfo.status === 0">
+        <button class="button" (click)="sendAcceptComment()">Accept</button>
+        <button class="button" (click)="sendRejectComment()">Reject</button>
+        <button class="button" (click)="sendCommentWithoutStatus()">Without status</button>      
+      </div>
+      <div class="comment-form__buttons" *ngIf="requestInfo.status === 1">
+          <button class="button" (click)="sendDoneComment()">Done</button>
+          <button class="button" (click)="sendRejectComment()">Reject</button>
+          <button class="button" (click)="sendCommentWithoutStatus()">Without status</button>      
+      </div>
+      <div class="comment-form__buttons" *ngIf="requestInfo.status !== 0 && requestInfo.status !== 1">
+          <button class="button" (click)="sendCommentWithoutStatus()">Without status</button>      
+      </div>
+    </div>
+    <div class="comments__comment comment" *ngFor="let comment of requestInfo.comments">
+      <p class="comment__title-text">Posted by <b class="comment__title-info">{{comment.commentOwner}}</b> at <b class="comment__title-info">{{comment.postDate | date:'short'}}</b></p>
```

Comment:
Using `<b>` is a bad practice. Use span instead and wrap strings on the next lines